secrets/secrets-kubernetes/go.mod

```diff
@@ -0,0 +1,61 @@
+module github.com/wasmCloud/wasmCloud-contrib/secrets/secrets-kubernetes
+
+go 1.22.5
+
+require (
+	github.com/golang-jwt/jwt v3.2.2+incompatible
+	github.com/golang-jwt/jwt/v5 v5.2.1
+	github.com/nats-io/nats-server/v2 v2.10.18
+	github.com/nats-io/nats.go v1.36.0
+	github.com/nats-io/nkeys v0.4.7
+	k8s.io/api v0.30.1
+	sigs.k8s.io/controller-runtime v0.18.4
```

Comment:
Is there any reason to prefer `sigs.k8s.io/controller-runtime` over plain `k8s.io/client-go`?

It's not that it's really a big deal, just seems like `controller-runtime` likely brings in a bunch of extra dependencies that aren't really necessary

Reply:
makes sense!